README.md

```diff
@@ -1,16 +1,19 @@
-# Title
+# Introduction to Class Variables and Methods in Ruby
 
-## Learning Goals
+Classes are like the skeleton from which individual objects are created and
```

Comment:
This is a good start, but since I think this sentence can be split - the first concept, classes are skeletons, is a good intro. Maybe expand a little on this - students should have only seen instance methods and instance variables so far - _now_, in this brick, we're introducing class variables and methods. 

I would reorder this a bit -

Classes are like the skeleton from which individual objects are created

So far, we've primarily created instance methods and instance variables for our individual objects to use. However, we can also create _class_ methods and variables. Class variables are used to store values related to a class rather than a particular instance. Class methods are
similarly used for implementing behavior that is related to a class in general rather than an instance.

(Something like that..)